src/Categories/Category/Construction/Quivers.agda

```diff
@@ -0,0 +1,191 @@
+        in begin
+          F₁ i f ▸ trans (F₀≡ eq) (F₀≡ eq′)  ≡˘⟨ ▸-assocʳ (F₁ i f) (F₀≡ eq′) ⟩
+          (F₁ i f ▸ F₀≡ eq) ▸ F₀≡ eq′        ≈⟨ ▸-resp-≈ (F₁≡ eq) (F₀≡ eq′) ⟩
+          (F₀≡ eq ◂ F₁ j f) ▸ F₀≡ eq′        ≡˘⟨ ◂-▸-comm (F₀≡ eq) (F₁ j f) (F₀≡ eq′) ⟩
+          F₀≡ eq ◂ (F₁ j f ▸ F₀≡ eq′)        ≈⟨ ◂-resp-≈ (F₀≡ eq) (F₁≡ eq′) ⟩
+          F₀≡ eq ◂ (F₀≡ eq′ ◂ F₁ k f)        ≡⟨ ◂-assocˡ (F₀≡ eq) (F₁ k f) ⟩
+          trans (F₀≡ eq) (F₀≡ eq′) ◂ F₁ k f  ∎
+      }
+    }
+  ; ∘-resp-≈  = λ {_ G H} {f g h i} eq eq′ → record
+    { F₀≡ = trans (cong (F₀ f) (F₀≡ eq′)) (F₀≡ eq)
+    ; F₁≡ = λ {_ _ j} →
+      let open EdgeReasoning H
+          open Transport (_⇒_ H)
+          open TransportOverQ (_⇒_ H) (_≈_ H)
+          open Transport (_⇒_ G) using () renaming (_▸_ to _▹_; _◂_ to _◃_)
+          open TransportMor (_⇒_ G) (_⇒_ H)
+          e₁ = F₀≡ eq
+          e₂ = F₀≡ eq′
+      in begin
+        F₁ (f QM.∘ h) j ▸ trans (cong (F₀ f) e₂) e₁ ≡˘⟨ ▸-assocʳ (F₁ f (F₁ h j)) e₁ ⟩
+        F₁ f (F₁ h j) ▸ cong (F₀ f) e₂ ▸ e₁         ≡˘⟨ cong (_▸ e₁) ( M-resp-▸ (F₀ f) (F₁ f) (F₁ h j) e₂) ⟩
+        F₁ f (F₁ h j ▹ e₂) ▸ e₁                     ≈⟨ F₁≡ eq ⟩
+        e₁ ◂ F₁ g (F₁ h j ▹ e₂)                     ≈⟨ ◂-resp-≈ e₁ (F-resp-≈ g (F₁≡ eq′)) ⟩
+        e₁ ◂ F₁ g (e₂ ◃ F₁ i j)                     ≡⟨ cong (e₁ ◂_) (M-resp-◂ (F₀ g) (F₁ g) e₂ (F₁ i j) ) ⟩
+        e₁ ◂ cong (F₀ g) e₂ ◂ F₁ g (F₁ i j)         ≡⟨ ◂-assocˡ e₁ (F₁ g (F₁ i j)) ⟩
+        trans e₁ (cong (F₀ g) e₂) ◂ F₁ g (F₁ i j)   ≡˘⟨ cong (_◂ F₁ g (F₁ i j)) (naturality (λ _ → e₁)) ⟩
+        trans (cong (F₀ f) e₂) e₁ ◂ F₁ g (F₁ i j)   ∎
+    }
+  }
+  where
+    open Quiver
+    open Morphism
+    open _≃_
+
+-- We can now build a forgetful (underlying) functor from categories to quivers
+Underlying₀ : Category o ℓ e → Quiver o ℓ e
+Underlying₀ C = record { Category C }
+
+Underlying₁ : {C : Category o ℓ e} {D : Category o′ ℓ′ e′} → Functor C D → Morphism (Underlying₀ C) (Underlying₀ D)
+Underlying₁ F = record { Functor F }
+
+Underlying : Functor (Cats o ℓ e) (Quivers o ℓ e)
+Underlying = record
+  { F₀ = Underlying₀
+  ; F₁ = Underlying₁
+  ; identity = λ {A} → record { F₀≡ = refl ; F₁≡ = Category.Equiv.refl A }
+  ; homomorphism = λ where {Z = Z} → record { F₀≡ = refl ; F₁≡ = Category.Equiv.refl Z }
+  ; F-resp-≈ = λ {A} {B} {F} {G} F≈G → record
+    { F₀≡ = λ {X} → eq₀ F≈G X
+    ; F₁≡ = λ {x} {y} {f} →
+      let open Category B
+          open HId B
+          UB = Underlying₀ B
+          open Transport (Quiver._⇒_ UB)
+          open Functor
+          open Quiver.EdgeReasoning (Underlying₀ B)
+      in begin
+        F₁ F f ▸ eq₀ F≈G y         ≈⟨ hid-subst-cod (F₁ F f) (eq₀ F≈G y) ⟩
+        hid (eq₀ F≈G y) ∘ F₁ F f   ≈⟨ eq₁ F≈G f ⟩
+        F₁ G f ∘ hid (eq₀ F≈G x)   ≈˘⟨ hid-subst-dom (eq₀ F≈G x) (F₁ G f) ⟩
+        eq₀ F≈G x ◂ F₁ G f         ∎
+    }
+  }
+  where
+  open NaturalTransformation
+  open NaturalIsomorphism
+  open _≡F_
+
+-- define these ahead of time
```

Comment:
These feel out of place here. Should they go into `Data.Quiver.Morphism`?

Reply:
Done.